de.nicokimmel.shadowcast-electron.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Name=shadowcast-electron
+Comment=Shows the videostream of ShadowCast by Genki
+Keywords=ShadowCast;Genki;Gaming
+Exec=/usr/bin/flatpak run de.nicokimmel.shadowcast-electron
```

Comment:
this is incorrect. it should match `command` more or less arguments.

Reply:
> this is incorrect. it should match `command` more or less arguments.

I was not quite sure about the desktop file. How do I fix this?

Reply:
Exec should just the main executable of your program, not the absolute path and definitely not `flatpak`.